### PR TITLE
chore(gh-pages): create redirect pages

### PR DESCRIPTION
**Pull request by Natalie-Smirnova**
The redirect links were crated for:
- ESL A11y Group
- ESLMediaQuery
- ESLTraversingQuery 
- ESLUtils

### CODE REVIEW

File: pages/views/redirect/esl-a11y-group.njk [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+layout: forward
+target: https://esl-ui.com/components/esl-a11y-group/
```
Comment (ala-n) [inline pages/views/redirect/esl-a11y-group.njk:3]: Should be relative.
We don't know if we going to change our domain + it should work locally with local domain

File: pages/views/_layouts/forward.njk [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="refresh" content="0;url={{ target }}"/>
+    <link rel="canonical" href="{{ target }}"/>
+    <title>Redirecting</title>
+</head>
+<body>
+    <p>This isn't here anymore!</p>
+    <p>Document has moved, if you aren't automatically redirected
+    <a href="{{ target }}">go here</a>.</p>
```
Comment (ala-n) [inline pages/views/_layouts/forward.njk:11]:
```suggestion
    <a href="{{ target | url }}">go here</a>.</p>
```
If I remember that correctly

File: pages/views/_layouts/forward.njk [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<html lang="en">
+<head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="refresh" content="0;url={{ target }}"/>
+    <link rel="canonical" href="{{ target }}"/>
+    <title>Redirecting</title>
```
Comment (ala-n) [inline pages/views/_layouts/forward.njk:6]:
```suggestion
    <title>Redirecting...</title>
```

File: pages/views/_layouts/forward.njk [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<html lang="en">
+<head>
```
Comment (ala-n) [inline pages/views/_layouts/forward.njk:2]: We also need to make this page with a site stylistic + favicon

File: pages/views/redirect/esl-traversing-query.njk [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
 layout: forward
-target: https://esl-ui.com/core/esl-traversing-query/
+target: /core/esl-traversing-query/
 permalink: /utils/esl-traversing-query/index.html
```
Comment (ala-n) [inline pages/views/redirect/esl-traversing-query.njk:4]:
```suggestion
permalink: /utils/esl-traversing-query.html
```

File: pages/views/redirect/esl-media-query.njk [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
 layout: forward
-target: https://esl-ui.com/core/esl-media-query/
+target: /core/esl-media-query/
 permalink: /utils/esl-media-query/index.html
```
Comment (ala-n) [inline pages/views/redirect/esl-media-query.njk:4]:
```suggestion
permalink: /utils/esl-media-query.html
```

File: pages/views/redirect/esl-a11y-group.njk [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
 layout: forward
-target: https://esl-ui.com/components/esl-a11y-group/
+target: /components/esl-a11y-group/
 permalink: /utils/esl-a11y-group/index.html
```
Comment (ala-n) [inline pages/views/redirect/esl-a11y-group.njk:4]:
```suggestion
permalink: /utils/esl-a11y-group.html
```

**Review by ala-n — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

File: pages/views/_layouts/base.njk [hunk 8]
```diff
@@ -2,7 +2,7 @@
 <html lang="{{ lang or 'en' }}" {% if scrollbar %} class="esl-scrollable-content" {% endif %}>
 <head>
   <title>{{ seoTitle or title or site.description }}</title>
-  <link rel="canonical" href="{{page.url | canonical | url }}"/>
+  <link rel="canonical" href="{{ canonicalURL or page.url  | canonical | url }}"/>
```
Comment (abarmina) [inline pages/views/_layouts/base.njk:5]: cosmetic:
```suggestion
  <link rel="canonical" href="{{ canonicalURL or page.url | canonical | url }}"/>
```

**Pull request merged**